orchagent/orch.cpp

```diff
@@ -853,6 +853,22 @@ task_process_status Orch::handleSaiCreateStatus(sai_api_t api, sai_status_t stat
                     exit(EXIT_FAILURE);
             }
             break;
+        case SAI_API_ROUTE:
+            switch (status)
+            {
+                case SAI_STATUS_SUCCESS:
+                    SWSS_LOG_WARN("SAI_STATUS_SUCCESS is not expected in handleSaiCreateStatus");
+                    return task_success;
+                case SAI_STATUS_ITEM_ALREADY_EXISTS:
```

Comment:
please also handle SAI_STATUS_ITEM_NOT_FOUND in handleSaiRemoveStatus

Reply:
Done